server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/binance/BinanceComExchange.java

```diff
@@ -36,6 +37,7 @@ public BinanceComExchange(String key, String secret, String preferredFiatCurrenc
     @Override
     public Set<String> getFiatCurrencies() {
         Set<String> fiatCurrencies = new HashSet<>();
+        fiatCurrencies.add(FiatCurrency.EUR.getCode());
```

Comment:
Please add here also USD

Reply:
But binance.com does not support USD (for now?), only a few USD stablecoins. We have binance.us (different class, different API url) that supports USD